src/main/java/catan/domain/model/game/DevelopmentCardsBean.java

```diff
@@ -0,0 +1,83 @@
+package catan.domain.model.game;
+
+import org.apache.commons.lang.builder.ToStringBuilder;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+import static org.apache.commons.lang.builder.ToStringStyle.SHORT_PREFIX_STYLE;
+
+@Embeddable
+public class DevelopmentCardsBean {
+
+    @Column(name = "KNIGHT", nullable = false)
+    private int knight;
+
+    @Column(name = "VICTORY_POINT", nullable = false)
+    private int victoryPoint;
+
+    @Column(name = "ROAD_BUILDING", nullable = false)
+    private int roadBuilding;
+
+    @Column(name = "MONOPOLY", nullable = false)
+    private int monopoly;
+
+    @Column(name = "YEAR_OF_PLENTY", nullable = false)
```

Comment:
Just thought that it is a good idea to name columns starting from some prefix, so that they will be placed close to each other in database, for example DEV_CARD_KNIGHT, DEV_CARD_VICTORY_POINT... 
The same approach should be applied to Resources